alluxio/alluxio_file_system.py

```diff
@@ -141,20 +143,32 @@ def __init__(
                 "Supply either 'etcd_hosts' or 'worker_hosts', not both"
             )
         self.logger = logger or logging.getLogger("AlluxioFileSystem")
+        if etcd_hosts and not worker_hosts:
+            self.logger.warning(
+                "Does not supply 'etcd_host'. Cannot tolerate Alluxio cluster changes."
+            )
         self.session = self._create_session(concurrency)
 
         # parse options
         page_size = ALLUXIO_PAGE_SIZE_DEFAULT_VALUE
+        hash_node_per_worker = int(ALLUXIO_HASH_NODE_PER_WORKER_DEFAULT_VALUE)
```

Comment:
why we put default value as string and convert here? We can just define as int at the beginning.

Reply:
@ChunxuTang previously put Alluxio related key=value string pair into the const file, we can discuss about what are the best way for dealing with property key and values